util/log/clog.go

```diff
@@ -37,6 +37,7 @@ import (
 	"time"
 
 	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/util"
```

Comment:
any package that `log` depends on can't use `log`. Can you move the functionality back into `log`?